### PR TITLE
VTCCA-4938 added if statement to handle no agents

### PR DESCRIPTION
**Pull request by zahiriqbal**

### CODE REVIEW

File: app/views/propertyrepresentation/manage/myAgents.scala.html [hunk 1]
```diff
@@ -30,36 +30,41 @@
...
-            ),
-            HeadCell(
-                content = Text(messages("propertyRepresentation.showAgents.label.action"))
-            )
-        )),
-        caption = None,
-        firstCellIsHeader = false
-    )
-    )
+  @if(agents.size > 1) {
```
[the remark this answers is not in the document]
Comment (darrell-plant) [reply app/views/propertyrepresentation/manage/myAgents.scala.html:34]: good point - better with something like "No agents assigned", better check with Martin

File: app/views/propertyrepresentation/manage/myAgents.scala.html [hunk 2]
```diff
@@ -30,36 +30,41 @@
...
+              HeadCell(
+                  content = Text(messages("propertyRepresentation.showAgents.label.action"))
+              )
+          )),
+          caption = None,
+          firstCellIsHeader = false
+      )
+      )
+  }else{
+      <p class="govuk-body">You have no agents.</p>
```
Comment (darrell-plant) [inline app/views/propertyrepresentation/manage/myAgents.scala.html:66]: add text messages file

**Comment by darrell-plant** [comment]
LGTDz 👍

**Comment by habeeb01** [comment]
LGTM 👍  - may be add a test as well?

**Pull request merged**